Documentation~/boolean.md

```diff
@@ -10,10 +10,11 @@ Each type of operation takes two ProBuilder Mesh objects as the left and right i
 
 You can only access this feature from the menu, because it is experimental:
 
-1. To open the Boolean (Experimental) window, navigate to Unity's top menu and go to **Tools** > **ProBuilder** > **Experimental** > **Boolean (CSG) Tool**.
-2. Set references to the ProBuilder Meshes just under the preview windows on the left and the right side.
-3. Select one of the boolean operation types from the **Operation** drop-down menu: [Intersection](#intersect), [Union](#union), or [Subtraction](#subtract).
-4. Click the **Apply** button. ProBuilder performs the selected operation.
+1. [Enable experimental features](preferences.md#experimental).
+2. To open the Boolean (Experimental) window, from the main menu, go to **Tools** > **ProBuilder** > **Experimental** > **Boolean (CSG) Tool**. 
+3. Set references to the ProBuilder Meshes just under the preview windows on the left and the right side.
+4. Select one of the boolean operation types from the **Operation** drop-down menu: [Intersection](#intersect), [Union](#union), or [Subtraction](#subtract).
```

Comment:
```suggestion
4. Select one of the boolean operation types from the **Operation** dropdown menu: [Intersection](#intersect), [Union](#union), or [Subtraction](#subtract).
```

Reply:
```suggestion
4. Select one of the boolean operation types from the **Operation** menu: [Intersection](#intersect), [Union](#union), or [Subtraction](#subtract).
```

Reply:
We're not really supposed to be too specific with UI components.